### ISSUE TITLE
rename library to bugware please...

### ISSUE THREAD

**Issue by charles-cs50** [beeware/briefcase#1620]
### Describe the bug

the library doesn't work if run a second time, the same 'android' project without any changes, it stays in a loop looking for the "pyproject.toml" that exists and is in the same place, without having been changed... making us angry ...

### Steps to reproduce

briefcase new android... command not work
briefcase new, work... it's ok, but not run...

[COMMENT REDACTED BY MAINTAINERS]

### Expected behavior

run android app... 

### Screenshots

run android project by command : briefcase run android -u

### Environment

- Operating System : Windows 10
- Python version : 3.11
- Software versions :
  - Briefcase : 0.3.16
  - Toga: "toga-android~=0.4.0"
  - ...

### Logs

```

```

### Additional context

It worked once and the next day, even though I created a new project, nothing happened... regrettable.

**Comment by freakboy3742** [comment]
I'm sorry to hear that you've had difficulty with Briefcase; but it's impossible to provide any assistance based on what you've provided here. Reports that say a command "doesn't work" doesn't help - Did you get an error message? Did it partially run and then crash? Was there a log file generated?

Reading between the lines of your report, I'm almost certain that the problem is user error - either not activating your virtual environment, or running Briefcase commands in a different directory. However, as you haven't provided any details, it's impossible to be certain or provide any more detail than that.

Furthermore, lashing out an accusing the maintainers of writing "bugware", or more nefarious intentions (which I've redacted) won't endear you to the people who are in a position to help you.

Closing on the basis that this isn't a bug report; it's a rant. If you'd like assistance, I'd suggest joining the Discord and asking a question, providing details about what you've done.
